Directory.Packages.props

```
@@ -1,8 +1,8 @@
   <Project>
   <PropertyGroup>
-    <EFCoreVersion>[9.0.0-rc.2.24460.3]</EFCoreVersion>
-    <MicrosoftExtensionsVersion>9.0.0-rc.2.24456.9</MicrosoftExtensionsVersion>
-    <NpgsqlVersion>8.0.4</NpgsqlVersion>
+    <EFCoreVersion>[9.0.0-rc.2.24474.1]</EFCoreVersion>
```

Why is EF Core version defined as an exact version?
I'm trying to upgrade to .NET 9 and am facing issues because it wants this **exact** version of EF to be installed.

```suggestion
<EFCoreVersion>9.0.0-rc.2.24474.1</EFCoreVersion>
```

---

Because there are frequently breaking changes between previews, and mixing preview versions is generally untested. Since EFCore.PG previews are frequently a bit delayed compared to the EF previews, people very frequently try to mix them together and open issues. I then have to close those issues and tell people to wait for the EFCore.PG preview version that corresponds to the EF one.

For the actual released version, an exact version match isn't required - it's OK to mix different patch versions from EFCore.PG and EF (but major version mixing still isn't allowed).